ui/css/treeherder-test-view.css

```diff
@@ -126,7 +126,7 @@ main .progress {
 }
 
 .toggle-count {
-  cursor: pointer;
+  background: transparent;
```

Comment:
There's a bootstrap utility you can use: `bg-transparent` :)

Reply:
Haha.  Got me again...  Thanks.  :)